public/viewjs/batteriesoverview.js

```diff
@@ -61,15 +61,15 @@ $(document).on('click', '.track-charge-cycle-button', function(e)
 	document.activeElement.blur();
 
 	Grocy.FrontendHelpers.BeginUiBusy();
-	
+
 	var batteryId = $(e.currentTarget).attr('data-battery-id');
 	var batteryName = $(e.currentTarget).attr('data-battery-name');
 	var trackedTime = moment().format('YYYY-MM-DD HH:mm:ss');
 
-	Grocy.Api.Get('batteries/track-charge-cycle/' + batteryId + '?tracked_time=' + trackedTime,
+	Grocy.Api.Post('batteries/' + batteryId + '/charged?tracked_time=' + trackedTime,
```

Comment:
Does this work? POST with query parameters is not typical. Data should be sent in POST body. POSTs will require changes on server side as well.